internal/loadbalancebenchmark/config.go

```diff
@@ -0,0 +1,120 @@
+package loadbalancingbenchmark
+
+import (
+	"fmt"
+	"time"
+)
+
+type ClientGroup struct {
+	Cnt    int
+	Rps    int
```

Comment:
In go, casing matters for style. Since rps is an initialism, it should be capitalized, eg `RPS`, similar to `ID` since this field must be public

Reply:
resolved.